go-controller/pkg/util/ndp/ra.go

```diff
@@ -0,0 +1,101 @@
+package ndp
+
+import (
+	"net"
+	"syscall"
+
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/mdlayher/socket"
+	"golang.org/x/sys/unix"
+)
+
+// RouterAdvertisement with mac, ips and lifetime field to send
+type RouterAdvertisement struct {
+	SourceMAC, DestinationMAC net.HardwareAddr
+	SourceIP, DestinationIP   net.IP
+	Lifetime                  uint16
+}
+
+// SendRouterAdvertisements will send variadic RAs.
+// Requires raw socket capabilities, since the RAs src MAC
+// and IP are not the ones on the LRP.
```

Comment:
logical router policy?
no
logical router port?
yes

Reply:
rewording, sinc eit does not make any sense.